README.md

```diff
@@ -3,3 +3,12 @@ Replacement for BeagleBone to handle readings from car's non-driverless sensors.
 
 `STM32CubeMX-app` application for setting up initial STM32 configuration
 `STM32F407VGTx` stuff related to our microcontroller - documents and CubeMX project file
+
+## Getting it working
+Utilizing Makefile in root directory. Help output is available, free to run `make` and read it.
+* Build all tools, firmware and flash: `make all`
+* Create docker images for building and flashing firmware: `make builder && make flasher`
+* Build firmware (given you have corresponding docker image): `make run_build`
```

Comment:
Suggestion: `make build_firmware` is more intuitive

Reply:
Maybe I should just change it to `make build` then?

Reply:
When I tested my STM32F4 Discovery Board, I re-activated the old software and compiled it with the newest GCC. Then, I flashed it to the board and could make it work (to test, just flash it and during the blinking blue LED, you can press the blue button; then, you can connect to the ChibiOS shell `screen /dev/ttyACM0` and interact with the software on the board).